x-pack/plugins/rule_registry/server/scripts/README.md

```diff
@@ -0,0 +1,24 @@
+Users with roles granting them access to monitoring (observability) and siem (security solution) should only be able to access alerts with those roles
```

Comment:
What are these scripts used for?

Reply:
@dhurley14 's used it as a quick way to test out RBAC via command line.

Reply:
This looks pretty useful, I'd love a walkthrough of how this works sometime (after this merges!)